resources/order.go

```diff
@@ -30,7 +30,7 @@ func (order *Order) Create(data map[string]interface{}, extraHeaders map[string]
 }
 
 // Edit updates an order having the given orderID.
-func (order *Order) Edit(orderID string, data map[string]interface{}, extraHeaders map[string]string) (map[string]interface{}, error) {
+func (order *Order) Update(orderID string, data map[string]interface{}, extraHeaders map[string]string) (map[string]interface{}, error) {
```

Comment:
Fix the comment on the line above this.
// Edit updates an order having the given orderID. => // Update updates an order having the given orderID.